app/index.js

```diff
@@ -2,50 +2,42 @@
 /**
  * @format
  */
-import 'react-native-gesture-handler'
```

Comment:
Are we updating the linter to use double quotes and different spacing?

Reply:
+1 for keeping it consistent with single quotes.

Reply:
This totally makes sense, I'll update this. Not sure why my vscode did that